aten/src/ATen/test/quantized_test.cpp

```diff
@@ -0,0 +1,41 @@
+#include <gtest/gtest.h>
+
+#include <ATen/ATen.h>
+#include <iostream>
+#include <limits>
+#include <sstream>
+#include <cmath>
+#include <type_traits>
+#include <ATen/test/test_assert.h>
+#include <ATen/Quantizer.h>
+
+using namespace at;
+
+TEST(TestQTensor, First) {
+  auto num_elements = 10;
+  Tensor r = at::ones({num_elements});
+  const float scale = 1.0;
+  const int32_t zero_point = 2;
+  Tensor qr = r.quantize_linear(scale, zero_point);
```

Comment:
Nit: Can we move to an API that explicitly creates a quantizer object and then passes it?
quantizer('quantizer_type')
qr =r.quantize(quantizer, qparams)
rqr =qr.dequantize(quantizer) # qparams are stored in qr as it is a qtensor

Reply:
Yes, I have a discussion in my quip doc about this, could you take a look